shardingsphere-proxy/shardingsphere-proxy-backend/src/main/java/org/apache/shardingsphere/proxy/backend/text/distsql/ral/queryable/ConvertYamlConfigurationHandler.java

```diff
@@ -67,61 +64,75 @@ protected Collection<LocalDataQueryResultRow> getRows(final ContextManager conte
         } catch (final IOException ex) {
             throw new ShardingSphereException(ex);
         }
-        String convertedDistSQL = generateConvertedDistSQL(yamlConfig);
-        return Collections.singleton(new LocalDataQueryResultRow(convertedDistSQL));
+        Preconditions.checkNotNull(yamlConfig, String.format("Invalid yaml file `%s`", file.getName()));
```

Comment:
Consider replacing String.format with Preconditions' method with template.

Reply:
OK, thank you.